clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/spark/input/hdfs/VeniceHdfsInputPartitionReaderFactory.java

```diff
@@ -7,6 +7,7 @@
 import org.apache.spark.sql.connector.read.PartitionReaderFactory;
 
 
+// TODO: Explore FilePartitionReaderFactory
```

Comment:
nit: can you provide a bit more context here why we need to explore it or maybe we can just create an internal ticket to track the improvement?

Reply:
I think I just forgot to remove this. I was exploring on this front previously